### DISCUSSION TITLE
Materials of Apollo Implant for NP2

### DISCUSSION THREAD

**Discussion by vasilisazhukova** [Coen-Lab/chronic-neuropixels#13] [General]
Hello, 
I start my work with NP2.
I would be very grateful to you if you could explain me how you connect Thor labs posts to the constructer head. 
Also, is it possible to work with this implant if it was printed in epoxy?
I am thinking to make the constructer head in metal, at least its base, and add thread for Thor labs posts. Do you think it is worthwhile to do?

I would appreciate very much if you could share more images of Apollo implant construction including when it is attached to a stereotactic station. 

Thanks.

**Comment by pipcoen** [comment]
Hi. Thanks for your comment. Can you confirm what you mean by "printed in epoxy" with a link to the material? 

For the constructor head, please see the section of the README where we list materials. Specifically: "M3 20mm screws: For connecting Constructor Head to Thor Labs posts (3 total)". There is a link to the screws we use there (but any equivalent screws will suffice).

**Reply by vasilisazhukova** [reply]
The material https://www.3djake.com/phrozen/water-washable-resin-grey

Thank you. I read it. My question is not about screws or type of inserts. My question is how do excactly you place inserts on the constructor head?

**Comment by pipcoen** [comment]
For the resin material you linked, I imagine it would work, although keep in mind that I can't guarantee, and the weight will differ from ours. A lot will depend on the printer/service you use. But we also print in resin, and it works fine.

For the Constructor head, there are no inserts. You simply use the screws I mentioned to screw directly into the threads on the Thor Labs Posts.

**Reply by vasilisazhukova** [reply]
Thank you very much. 

Excuse me, I confused the details. I meant the base of NP2-Alpha_PayloadHolder.

**Reply by pipcoen** [reply]
This is connected via an M3 heated insert in the provided hole (circled in red here:

![image](https://github.com/Coen-Lab/chronic-neuropixels/assets/1191043/2324369c-3ea2-4478-b2c7-c02dd0c17db5)
